XamarinCommunityToolkit.UnitTests/ConvertersNamespace_Tests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Linq;
+using System.Reflection;
+using Xamarin.CommunityToolkit.Behaviors;
+using Xamarin.CommunityToolkit.Converters;
+using Xamarin.CommunityToolkit.Effects;
+using Xamarin.CommunityToolkit.Extensions;
+using Xamarin.Forms;
+using Xunit;
+
+namespace Xamarin.CommunityToolkit.UnitTests
+{
+	public class Namespace_Tests
+	{
+		[Fact]
+		public void MakeSureConvertersAreInTheRightNamespace()
+		{
+			var allTheTypes = Assembly.GetAssembly(typeof(InvertedBoolConverter)).GetTypes();
+
+			foreach (var type in allTheTypes.Where(t => t.Name.EndsWith("Converter") && t.GetInterface("IValueConverter") != null))
```

Comment:
Sorry, I don't want to be so picky, but I guess that we should use **nameof** as often as possible (for types and namespaces if it's possible)

e.g. nameof(IValueConverter), nameof(Xamarin.CommunityToolkit.Converters) etc.

Reply:
No worries, that's what reviews are for!

For the `IValueConverter` yes. For the whole namespace it won't work because it will only return to you the last bit. I.e. `nameof(Xamarin.CommunityToolkit.Converters` returns only `Converters` and thus the assert will fail